plugins/scripts/juno-pipeline-installer.sh

```diff
@@ -0,0 +1,12 @@
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+
+# app icon setup
+cd $SCRIPT_DIR
+cp "../assets/nuke.png" "$1/nuke.png"
```

Comment:
you can add our own nuke icon if you want, or ping me and ill create one to make a visual difference thats this is pipeline nuke ()we now have what 3-4 versions? ) let me know!

Reply:
Yes I was going to ping you on Monday if you were interested in making a Juno Nuke Icon! if you are up for it that be awesome!

Reply:
added nuke_juno.png icon. Named it that way so in our assets it's grouped with our other nuke icons vs being separated.